packages/storage/src/implementation/error.ts

```diff
@@ -221,15 +222,30 @@ export function noDownloadURL(): FirebaseStorageError {
   );
 }
 
+export function invalidArgument(message: string): FirebaseStorageError;
 export function invalidArgument(
   index: number,
   fnName: string,
   message: string
+): FirebaseStorageError;
```

Comment:
Note that there is only a single callsiste for `invalidArgument` with `fnName` and `message`. We could simplify this code by removing the outlier.

Reply:
Ack, missed removing that param type check.